pubtools/_quay/tag_docker.py

```diff
@@ -150,6 +150,50 @@ def verify_input_data(self):
             if item.metadata["tag_source"] and ":" in item.metadata["tag_source"]:
                 raise BadPushItem("Specifying source via digest is not allowed.")
 
+            # if prod target is selected, additional checks are made
```

Comment:
Can you please move this to another method which isn't called called in the __init__() but in run()? I don't think it's good idea to start querying quay immediately when instance is initiated

Reply:
Fixed